cms/djangoapps/models/settings/waffle.py

```diff
@@ -3,15 +3,14 @@
 """
 from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
 
+
 WAFFLE_NAMESPACE = 'grades'
 
 # edx/edx-platform feature
 MATERIAL_RECOMPUTE_ONLY = 'MATERIAL_RECOMPUTE_ONLY'
 
 MATERIAL_RECOMPUTE_ONLY_FLAG = CourseWaffleFlag(  # lint-amnesty, pylint: disable=toggle-missing-annotation
-    waffle_namespace=WAFFLE_NAMESPACE,
-    flag_name=MATERIAL_RECOMPUTE_ONLY,
-    module_name=__name__,
+    f'{WAFFLE_NAMESPACE}.{MATERIAL_RECOMPUTE_ONLY}', __name__
```

Comment:
We do not have a prefix here, should we add it?

Reply:
It's not necessary and I tried not to change the initial flow